src/main/java/org/commcare/modern/database/DatabaseHelper.java

```diff
@@ -34,6 +34,11 @@
     public static Pair<String, String[]> createWhere(String[] fieldNames, Object[] values,
```

Comment:
Can we add a test to confirm that the underlying endpoints that use this return all results and not an error, so the behavior can be regressed?

Reply:
Good point, does [this](https://github.com/dimagi/commcare-core/pull/930/commits/7cb5743d401602d9382cc2e8b8b242878b8b4985) look good ?